src/Types.ts

```diff
@@ -605,6 +605,8 @@ export interface ModalizerProps {
     isAlwaysAccessible?: boolean;
     isNoFocusFirst?: boolean;
     isNoFocusDefault?: boolean;
+    /** A focus trap variant, keeps focus inside the modal when tabbing */
+    circular?: boolean;
```

Comment:
I think we need a more speaking name here. Like trapFocus.

Reply:
or isFocusTrapped (to be like the other props here).

Reply:
named to `isTrapped`